keel-sql/src/main/resources/db/changelog/20200714-add-event-application-column.yml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+- changeSet:
+    id: add-event-application-column
+    author: fletch
+    changes:
+    - sql:
+        sql: |
+          alter table event add column application varchar(255) generated always as (json ->> '$.application') stored;
```

Comment:
Do you have a sense as to:
* how long this migration will take against main? (There are about 162K records in `event` right now)
* what will the impact be on the behavior of keel while the migration is happening? (Will this prevent keel from writing or reading the event table while it's happening?)

Reply:
Good question. Running a query to extract that data from the event table takes under a second so I don't think there should be any significant impact.